### PR TITLE
Epo ops 3.2

### PR DESCRIPTION
**Pull request by telzul**
version bump of epo ops api
update to running travis on current ruby versions
some minor changes

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.3.3
```
Comment (tamaloa) [inline .ruby-version]: as the gem targets several rubies (see .travis.yml) we should not include a .ruby-version file.

**Review by tamaloa — CHANGES_REQUESTED** [review]
Nice work.

Please add:
* Reference to compatible ruby versions to README (refer to travis.yml)
* minor version bump in preparation of new release

File: .ruby-version (hunk 1 above)
Comment (telzul) [inline .ruby-version]: yes, i removed the file and also referred to the .travis.yml file in readme

**Review by tamaloa — APPROVED** [review]

**Comment by tamaloa** [comment]
Let's `rake release` it :)

**Pull request merged**